src/policy/fees.h

```diff
@@ -159,6 +171,10 @@ class CBlockPolicyEstimator
 
 class FeeFilterRounder
 {
+private:
+    static constexpr double MAX_FILTER_FEERATE = 1e7;
+    static constexpr double FEE_FILTER_SPACING = 1.1;
```

Comment:
Can you add a comment to explain the relation with the CBlockPolicyEstimator::FEE_SPACE constant?